content/about/timeline.md

```diff
@@ -5,51 +5,32 @@
 
 > Note:
```

Comment:
does this need to be a note insde the `>`, this way it doesn't render the links as links. Could be just a normal text?